kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/expressions/util/UserActionCheckerTest.java

```diff
@@ -0,0 +1,905 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.workbench.common.dmn.client.editors.expressions.util;
+
+import com.ait.lienzo.test.LienzoMockitoTestRunner;
+import org.assertj.core.util.Arrays;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kie.workbench.common.dmn.client.editors.expressions.jsinterop.props.Annotation;
+import org.kie.workbench.common.dmn.client.editors.expressions.jsinterop.props.Clause;
+import org.kie.workbench.common.dmn.client.editors.expressions.jsinterop.props.Column;
+import org.kie.workbench.common.dmn.client.editors.expressions.jsinterop.props.ContextEntryProps;
+import org.kie.workbench.common.dmn.client.editors.expressions.jsinterop.props.ContextProps;
+import org.kie.workbench.common.dmn.client.editors.expressions.jsinterop.props.DecisionTableProps;
+import org.kie.workbench.common.dmn.client.editors.expressions.jsinterop.props.DecisionTableRule;
+import org.kie.workbench.common.dmn.client.editors.expressions.jsinterop.props.ExpressionProps;
+import org.kie.workbench.common.dmn.client.editors.expressions.jsinterop.props.InvocationProps;
+import org.kie.workbench.common.dmn.client.editors.expressions.jsinterop.props.RelationProps;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+@RunWith(LienzoMockitoTestRunner.class)
+public class UserActionCheckerTest {
+
+    private static final String NAME = "name";
+    private static final String AGGREGATION = "aggregation";
+    private static final String DATA_TYPE = "dataType";
+    private static final String HIT_POLICY = "hitPolicy";
+
+    private UserActionChecker checker;
+
+    @Before
+    public void setup() {
+        checker = spy(new UserActionChecker());
+    }
+
+    @Test
+    public void testIsUserAction_WhenIsDecisionTable() {
+
+        final DecisionTableProps decisionTableProps = new DecisionTableProps(NAME,
+                                                                             "data type",
+                                                                             "hit policy",
```

Comment:
it seems we forget to use DATA_TYPE and HIT_POLICY on some places in this test class?